react/__tests__/components/SKUSelector.test.tsx

```diff
@@ -1121,7 +1123,7 @@ describe('<SKUSelector />', () => {
         skuSelected={skuItems[0]}
         skuItems={skuItems}
         maxItems={6}
-        showValueNameForImageVariation={true}
+        showValueNameForImageVariation
```

Comment:
Is this change okay? Isn't the default value `undefined`?

Reply:
No, the default value is `true`